### PR TITLE
ceph: fix external script when passing monitoring list

### PR DESCRIPTION
**Pull request by leseb**
**Description of your changes:**

If we call the script with --monitoring-endpoint
10.1.8.29,10.1.8.36,10.1.8.25, we need to parse each comma-separated
address and not the entire block.
Now each address is tested properly.

Signed-off-by: Sébastien Han <seb@redhat.com>

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**

- [ ] **Commit Message Formatting**: Commit titles and messages follow guidelines in the [developer guide](https://rook.io/docs/rook/latest/development-flow.html#commit-structure).
- [ ] **Skip Tests for Docs**: Add the flag for skipping the build if this is only a documentation change. See [here](https://github.com/rook/rook/blob/master/INSTALL.md#skip-ci) for the flag.
- [ ] **Skip Unrelated Tests**: Add a flag to run tests for a specific storage provider. See [test options](https://github.com/rook/rook/blob/master/INSTALL.md#test-storage-provider).
- [ ] Reviewed the developer guide on [Submitting a Pull Request](https://rook.io/docs/rook/latest/development-flow.html#submitting-a-pull-request)
- [ ] Documentation has been updated, if necessary.
- [ ] Unit tests have been added, if necessary.
- [ ] Integration tests have been added, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.

### CODE REVIEW

File: cluster/examples/kubernetes/ceph/create-external-cluster-resources.py [hunk 1]
```diff
@@ -403,15 +403,18 @@ def get_active_and_standby_mgrs(self):
             monitoring_endpoint_port = self.DEFAULT_MONITORING_ENDPOINT_PORT
```
Comment (aruniiird) [inline cluster/examples/kubernetes/ceph/create-external-cluster-resources.py:404]: If we can add the following lines before `try` (that is before line#405), we could completely avoid any other further changes (from line#406 on wards). 

```python
        # user could give comma and space separated inputs (like --monitoring-endpoint="<ip1>, <ip2>")
        monitoring_endpoint_ip_list = monitoring_endpoint_ip_list.replace(",", " ")
        monitoring_endpoint_ip_list_split = monitoring_endpoint_ip_list.split()
        # if monitoring-endpoint could not be found, raise an error
        if len(monitoring_endpoint_ip_list_split) == 0:
            raise ExecutionFailureException("No 'monitoring-endpoint' found")
        # first ip is treated as the main monitoring-endpoint
        monitoring_endpoint_ip = monitoring_endpoint_ip_list_split[0]
        # rest of ip-s are add to the 'standby_mgrs'
        standby_mgrs.extend(monitoring_endpoint_ip_list_split[1:])
```

Comment (leseb) [reply cluster/examples/kubernetes/ceph/create-external-cluster-resources.py:405]: That works too! Thanks!

**Review by aruniiird — APPROVED** [review]
Changes look good...

File: cluster/examples/kubernetes/ceph/create-external-cluster-resources.py [hunk 2]
```diff
@@ -394,14 +395,26 @@ def get_active_and_standby_mgrs(self):
...
+        # user could give comma and space separated inputs (like --monitoring-endpoint="<ip1>, <ip2>")
+        monitoring_endpoint_ip_list = monitoring_endpoint_ip_list.replace(
+            ",", " ")
+        monitoring_endpoint_ip_list_split = monitoring_endpoint_ip_list.split()
+        # if monitoring-endpoint could not be found, raise an error
+        if len(monitoring_endpoint_ip_list_split) == 0:
+            raise ExecutionFailureException("No 'monitoring-endpoint' found")
+        # first ip is treated as the main monitoring-endpoint
+        monitoring_endpoint_ip = monitoring_endpoint_ip_list_split[0]
+        # rest of ip-s are add to the 'standby_mgrs'
```
Comment (aruniiird) [inline cluster/examples/kubernetes/ceph/create-external-cluster-resources.py:415]:
```suggestion
        # rest of the ip-s are added to the 'standby_mgrs' list
```

**Pull request merged**